### ISSUE TITLE
Find The Builders

### ISSUE THREAD

**Issue by RobHowdle** [Droid-Builder-Web-Team/db_mot#80]
Ability to have a "beacon" in every venue. Using the events app you can use a form of google maps to track the becon in realtime allowing you to find the Droid Builders Stand in a large event. Especially useful for people who are new to the event/ building and do not know the names of rooms/ buildings.

**Comment by RobHowdle** [comment]
Still want to work on this as a nice extra for any large comic con style events. Will research into device tracking.

**Comment by dpoulson** [comment]
Yeah, its a great idea. Just think its a better one for the main builders site. 

Events like MCM we could 'light the beacon' and let the public use it to find the builders.

**Comment by RobHowdle** [comment]
🤣 I was imaging the same thing! We could have a GPS tracker of sorts and if we have a physical device that we could track we could then do something separate and enclose this device in a form of beacon. A modal of Lee posing like the Statue of Libraty with a droid Builder Guide in one arm, a lightsaber in the other raised up and the beacon making the saber glow! Although with something like that I don't know why you'd need a beacon in the first place 😆
